Documentation/kubernetes.md

```diff
@@ -84,14 +131,14 @@ $ kubectl create secret \
     --from-literal=client-secret=$GITHUB_CLIENT_SECRET
 ```
 
-Create the dex deployment, configmap, and node port service.
+### Deploy the Dex server
+
+Create the dex deployment, configmap, and node port service. This will also create RBAC bindings allowing the Dex pod access to manage [Custom Resource Definitions](https://github.com/dexidp/dex/blob/master/Documentation/storage.md#kubernetes-custom-resource-definitions-crds) within Kubernetes.
```

Comment:
Could you try making this a relative link instead? Maybe this could work:
```suggestion
Create the dex deployment, configmap, and node port service. This will also create RBAC bindings allowing the Dex pod access to manage [Custom Resource Definitions](storage.md#kubernetes-custom-resource-definitions-crds) within Kubernetes.
```

Reply:
Thanks for catching that! I've updated it.